src/lib/cloud-metadata/README.md

```diff
@@ -128,7 +138,7 @@ You can run this example `manifest` by saving it as `./examples/manifests/test/c
 ```sh
 npm i -g @grnsft/if
 npm i -g @grnsft/if-plugins
-ie --manifest ./examples/manifests/test/cim.yml --output ./examples/outputs/cim.yml
+ie --manifest ./examples/manifests/test/cim.yml --output ./examples/outputs/cim
```

Comment:
@jmcook1186 As cim.yml was removed, maybe will be better to see a real manifest name here, like cloud-metadata.yml

Reply:
I've brought cim.yml back by renaming it to cloud-metadata.yml in my PR. @jmcook1186 could you rename `cim` here to `cloud-metadata`?

Reply:
```suggestion
ie --manifest ./examples/manifests/test/cloud-metadata.yml --output ./examples/outputs/cloud-metadata
```